govcd/user.go

```diff
@@ -71,9 +71,9 @@ func NewUser(cli *Client, org *AdminOrg) *OrgUser {
 	}
 }
 
-// FetchUserByHref returns a user by its HREF, without need for
+// GetUserByHref returns a user by its HREF, without need for
 // searching in the adminOrg user list
-func (adminOrg *AdminOrg) FetchUserByHref(href string) (*OrgUser, error) {
```

Comment:
Isn't  `FetchUserByHref` was already release and we have to deprecate?

Reply:
yes. Fixed